artifact/metadata.go

```diff
@@ -346,17 +367,36 @@ func (t TypeInfoProvides) Map() map[string]interface{} {
 }
 
 func NewTypeInfoProvides(m interface{}) (ti TypeInfoProvides, err error) {
+
+	const errMsgInvalidTypeFmt = "Invalid TypeInfo provides type: %T"
```

Comment:
Question: Is it really necessary to pull these out when they are used in one place only?

Reply:
It was done strictly out of formatting preference, it felt awkward to insert the write the string in-place and not violate the 80 character column width rule ;)
btw, this const is indirectly referenced two times as well.